apps/web/src/server/views/lpa/check-and-confirm.njk

```diff
@@ -0,0 +1,48 @@
+{% extends "_layouts/layout-base.njk" %}
+
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block header %}
+	{{ govukHeader({
+		homepageUrl: "https://www.gov.uk/",
+		serviceName: "Appeal a planning decision",
+		serviceUrl: "/lpa",
+		containerClasses: "govuk-width-container"
+	}) }}
+{% endblock %}
+
+{% block content %}
+	{{ super() }}
+
+	<h1 class="govuk-heading-xl">
+		Review questionnaire
```

Comment:
You also need to set the `{% block pageTitle %}` else this heading will not appear in the browser tab

Reply:
I've added this to the 3 existing LPA templates